pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +550,64 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(graph, start, path_length, seed = 0, alpha = None):
+    """
+    Executes random walk from the starting node and given graph.
+
+    Parameters
+    ==========
+
+    graph: Graph
+        The graph on which random walk needs to be executed.
+    start: GraphNode or it's subclasses
+        The node/vertex from where random walk will begin.
+    path_length:
+        The maximum number of nodes/vertices traversed by random walk.
+    alpha:
+        A parameter which determines the randomness. More the value more the chances of restart.
+
+    Returns
+    =======
+
+    path: list
+        Python list of nodes after execution of random walk.
+
+    Examples
+    ========
+
+    >>> from pydatastructs import Graph, AdjacencyListGraphNode
+    >>> from pydatastructs import random_walk
+    >>> v1, v2, v3, v4, v5  = [AdjacencyListGraphNode(i) for i in range(5)]
+    >>> g = Graph(v1, v2, v3, v4, v5)
+    >>> g.add_edge(v1.name, v2.name)
+    >>> g.add_edge(v2.name, v3.name)
+    >>> g.add_edge(v3.name, v1.name)
+    >>> g.add_edge(v2.name, v4.name)
+    >>> g.add_edge(v4.name, v5.name)
+
+    >>> random_walk = random_walk(g, v1, 3)
+    >>> random_walk == ['v1', 'v2', 'v5']
+    True
+
+    """
+
+    path = [start.name]
+
+    for i in range(0, path_length):
+        cur = path[-1]
+        degree = len(graph.neighbors(cur))
```

Comment:
```suggestion
```

Reply:
Everywhere else in the code the usage is the same. Why the error is here?